gpu4pyscf/solvent/pcm.py

```diff
@@ -393,19 +394,23 @@ def _get_v(self, dms):
         return electrostatic potential on surface
         '''
         nset = dms.shape[0]
-        ngrids = self.surface['grid_coords'].shape[0]
+        charge_exp  = self.surface['charge_exp']
+        grid_coords = self.surface['grid_coords']
+        ngrids = grid_coords.shape[0]
         v_grids_e = cupy.empty([nset, ngrids])
         for i in range(nset):
-            v_grids_e[i] = 2.0*int3c2e.get_j_int3c2e_pass1(self.intopt, dms[i])
+            v_grids_e[i] = intor(self.mol, "int1e_grids", grid_coords, dm = dms[i], charge_exponents = charge_exp**2, intopt = self.intopt)
```

Comment:
Move the loop into `intor` and support multiple density matrices. We could possibly improve the efficiency by batching density matrices.